RestaurantReservation/Services/MenuItmesServices/MenuItemsServices.cs

```diff
@@ -0,0 +1,45 @@
+using RestaurantReservation.Db;
+using RestaurantReservation.Models;
+using Microsoft.EntityFrameworkCore;
+namespace RestaurantReservation.Services.MenuItemsServices
+{
+    public class MenuItemsService : IMenuItemsService 
+    {
```

Comment:
This should be a repository, not a service, you are reversing the logic. 
The repository layer should talk with the database and the service layer should talk with the repository layer, so you need to update all service and repository layer